### PR TITLE
Pull request for mysqli

### PR DESCRIPTION
**Pull request by dgatwood**
Updates code to use mysqli and parameterized queries throughout.

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -1,35 +1,14 @@
-RewriteEngine On
+# AddOutputFilter INCLUDES .html
```
Comment (seanjohnson08) [inline .htaccess]: Please remove `.htaccess` from your diff? I honestly shouldn't even have it in the repo and will remove it soon.

File: Service/jsrun.js [hunk 2]
```diff
@@ -2,6 +2,17 @@ var updatetime=5000;
 var useJSLinks=1;
 var adcounter=0;
 
+/* Returns the path to this script. */
+function getJXBDBaseDir() {
+    var elts = document.getElementsByTagName("script");
+    for (var i=0; i < elts.length; i++) {
+	var elt = elts[i];
+	// console.log("Script at:"+elt.src+"\n");
```
Comment (seanjohnson08) [inline Service/jsrun.js]: remove console log

File: Service/jsrun.js [hunk 3]
```diff
@@ -380,9 +391,12 @@ function RUNF(){
  if(useJSLinks&&document.location.toString().indexOf('?')>0) document.location=(useJSLinks==2?'':'./')+"#"+document.location.search.substr(1)+(document.location.hash?document.location.hash:'');
 
  if(Sound) {
-  Sound.load("sbblip","http://jaxboards.com/Sounds/blip.mp3",false);
-  Sound.load("imbeep","http://jaxboards.com/Sounds/receive.mp3",false);
-  Sound.load("imnewwindow","http://jaxboards.com/Sounds/receive.mp3",false);
+  basedir = getJXBDBaseDir();
+
+  // console.log("BASEDIR: "+basedir+"\n");
```
Comment (seanjohnson08) [inline Service/jsrun.js]: remove

File: crossdomain.xml [hunk 4]
```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0"?>
```
Comment (seanjohnson08) [inline crossdomain.xml]: this is another file I should remove from the repo, it doesn't belong

File: config.php [hunk 5]
```diff
@@ -1,14 +1,20 @@
-<?
+<?php
 $CFG=Array(
+/* "mail_from"=>"jaxboards.com <no-reply@jaxboards.com>", */
+"mail_from"=>"WebScriptedTV.com <no-reply@webscriptedtv.com>",
```
Comment (seanjohnson08) [inline config.php]: please update config file to be more generic (remove your sql credentials) and remove "WebScriptedTV" references

File: Service/jsrun.js (hunk 3 above)
Comment (dgatwood) [inline Service/jsrun.js]: Done.

File: Service/jsrun.js (hunk 2 above)
Comment (dgatwood) [inline Service/jsrun.js]: Done.

File: .htaccess (hunk 1 above)
Comment (dgatwood) [inline .htaccess]: Done.

**Comment by dgatwood** [comment]
No prob.  I wanted a properly functioning board that used AJAX calls so that I could integrate it with a completely different authentication system that only supported AJAX, and I had previously encountered so much trouble with PHPBB and security holes that I decided whatever I implemented, I wanted to audit the heck out of it (hence, the giant PR).  :-)

**Pull request merged**